src/entity/Game.ts

```diff
@@ -33,12 +56,48 @@ export class Game {
     @Property({ nullable: true })
         encounterRound?: number;
 
-    constructor(startcardID: number) {
-        this.startcard = Reference.createFromPK(Card, startcardID);
+    constructor(id:string, startcard: Card) {
+        this.id = id;
+        this.startcard = startcard;
     }
-}
 
-export enum GameRole {
-    ROUND_ROBIN,
-    SUDDEN_DEATH
+    startGame() {
+        throw new Error('Method not implemented. Game to Start: ' + this.id);
+    }
+
+    /**
+     *
+     * @param freeSockets list of all sockets to use for the players
+     */
+    public async populatePlayerSockets(sockets: NopeSocket[]) : Promise<boolean> {

```

Comment:
Hier verstehe ich die Funktion nicht richtig. Werden hier alle Sockets der Spieler zurückgegeben, die gerade frei sind? Der Kommentar `list of all sockets to use for the players` ist für mich nicht eindeutig genug.